apps/remote_control/lib/lexical/remote_control/code_action/handlers/organize_aliases.ex

```diff
@@ -37,9 +40,32 @@ defmodule Lexical.RemoteControl.CodeAction.Handlers.OrganizeAliases do
   end
 
   defp check_aliases(%Document{}, %Analysis{} = analysis, %Range{} = range) do
-    case Analysis.module_scope(analysis, range) do
-      %Scope{aliases: [_ | _]} -> :ok
-      _ -> :error
+    with %Scope{aliases: [_ | _]} <- Analysis.module_scope(analysis, range),
+         true <-
+           ancestor_is_alias?(analysis, range.start) or
+             token_at_cursor_is_alias?(analysis, range.start) do
+      :ok
+    else
+      _ ->
+        :error
     end
   end
+
+  defp ancestor_is_alias?(%Analysis{} = analysis, %Position{} = position) do
+    analysis
+    |> Ast.cursor_path(position)
+    |> Enum.any?(fn
+      {:alias, _, _} ->
+        true
+
+      _ ->
+        false
+    end)
+  end
+
+  defp token_at_cursor_is_alias?(%Analysis{} = analysis, %Position{} = position) do
+    project = RemoteControl.get_project()
+    {:ok, env} = Ast.Env.new(project, analysis, position)
```

Comment:
Is this guaranteed to return `{:ok, env}`? If it doesn't, this function should probably just return `false` instead of crashing.